hrs-portlets-webapp/src/main/java/edu/wisc/portlet/hrs/web/timeabs/TimeSheetDataController.java

```diff
@@ -48,15 +51,32 @@ public void setTimeSheetDao(TimeSheetDao timeSheetDao) {
     }
 
     
+    /**
+     * Gets Time sheets - limited to most recent 80 results
+     * @param modelMap
+     * @return the most recent timesheets, limited to 80
+     */
```

Comment:
Nicely done on the JavaDoc.

Maybe "Gets most recent up to 80 TimeSheets for the requesting user."